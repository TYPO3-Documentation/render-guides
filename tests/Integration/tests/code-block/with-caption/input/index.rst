============
With caption
============

..  code-block:: php
    :caption: src/hello.php

    echo 'Hello, TYPO3';
