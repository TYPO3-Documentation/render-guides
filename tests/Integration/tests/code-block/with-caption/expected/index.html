<!-- content start -->
                <section class="section" id="with-caption">
            <h1>With caption<a class="headerlink" href="#with-caption" title="Permalink to this headline">Â¶</a></h1>

                    <div class="code-block-caption">
            <span class="caption-text">src/hello.php</span>
        </div><div class="code-block-wrapper" translate="no">

        <pre class="code-block"><code class="hljs php"><span class="hljs-keyword">echo</span> <span class="hljs-string">'Hello, TYPO3'</span>;</code></pre>
        <button class="code-block-copy" title="Copy to clipboard"><svg class="code-block-copy-icon" width="24" height="24"><title>Copy to clipboard</title><rect x="8" y="8" width="12" height="12" rx="2"></rect><path d="M16 8v-2a2 2 0 0 0 -2 -2h-8a2 2 0 0 0 -2 2v8a2 2 0 0 0 2 2h2"></path></svg></button>
    </div>
    </section>

        <!-- content end -->
